.github/CODEOWNERS

```diff
@@ -670,6 +670,7 @@ playwright.config.ts @grafana/plugins-platform-frontend
 # Kind definitions
 /kinds/dashboard @grafana/dashboards-squad
 /kinds/ @grafana/grafana-as-code
+kindsv2/ @grafana/dashboards-squad
```

Comment:
Just wondering about the structure here. We already have kinds under `kinds/dashboard`. Shouldn't `kindsv2` live under that path and not in the root? Do we know if other features will be under kindsv2 or just dashboard?